src/tomlize/main.py

```diff
@@ -14,14 +14,14 @@ def main(argv):
     coloredlogs.install(level="INFO", fmt="%(message)s")
     output_file = pathlib.Path("pyproject.toml")
 
-    existing_config = None
+    existing_config = tomlkit.TOMLDocument()
```

Comment:
We might want to not create an object here if we end up reassigning the variable.